src/Axe/Data/AxeDbDeployment.cs

```diff
@@ -45,6 +45,7 @@ public static class AxeDbDeployment
                 InformationText =
 @"C# is a programming language that is designed for building a variety of applications that run on the .NET Framework.
 C# is simple, powerful, type-safe, and object-oriented",
+                Template = "using System;\nnamespace Axe\n{\npublic class AxeTask\n{\npublic int Main()\n{\n}\n}\n}",
```

Comment:
`Template` string isn't readable with all those \n
I suggest to use `@""` string:
`@"using System;
namespace Axe 
{
...
"`

Reply:
It's temporary solution because this template stores saves to db only and then is used as just string. Special symbols provides comfortable showing in code editor.